### ISSUE TITLE
Could you please add a license?

### ISSUE THREAD

**Issue by draffensperger** [leafgarland/typescript-vim#128]
This would make it clearer how the software can be used legally - perhaps the Vim license or the MIT license.

**Comment by leafgarland** [comment]
Sure, although the original version of the syntax file came from Microsoft and the indent source is mostly copied from https://github.com/pangloss/vim-javascript, so it might be a bit messy.

**Comment by draffensperger** [comment]
Looks like the indent source has the Vim license, though that doesn't answer the Microsoft contribution piece.

**Comment by empijei** [comment]
Gentle bump on this, I'd really like to use this at work but the lack of licensing makes this impossible :cry:

**Comment by leafgarland** [comment]
Hi @empijel, I appreciate the gentle bump but the fact that the original source for parts of this repo is Microsoft and was not licensed makes it hard, if not impossible, to license. We would need to rewrite. 

This has already prevented this code from being put in the base vim distribution and it is likely we will see another typescript config become part.

**Comment by draffensperger** [comment]
Would it make sense to add something in the README about this intentional lack of license and pointing people to an alternative if that is a blocker for them? For instance, could we point people to https://github.com/HerringtonDarkholme/yats.vim, which has the Vim license?

**Comment by leafgarland** [comment]
Yes now that yats repo is licensed and is likely to be part of the vim distribution retiring this repo or at least notifying in the readme is a good idea.
